app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -34,12 +34,24 @@ def rows
         consent_row,
 
       ].flatten.compact
+
+      base_rows.insert(6, confidentiality_row) if feature_active_and_feedback_provided?
```

Comment:
```suggestion
      base_rows.insert(6, confidentiality_row) if FeatureFlag.active?(:show_reference_confidentiality_status)
```

Reply:
I've gone with this: https://github.com/DFE-Digital/apply-for-teacher-training/pull/10137/commits/6c6b15577de54c3db7ae112254f0979b434ba40f